pkg/api/apiv1/apiv1.go

```diff
@@ -37,6 +38,8 @@ type Opts struct {
 	JobQueueReader queue.JobQueueReader
 	// CancellationReadWriter reads and writes cancellations to/from a backing store.
 	CancellationReadWriter cqrs.CancellationReadWriter
+	// QueueShardSelector determines the queue shard to use
+	QueueShardSelector redis_state.ShardSelector
```

Comment:
`QueueShardSelector` is not added added in the dev or start commands for the CLI. Should it be? It looks like `AddRoutes` only handles `AuthFinder` or `AuthMiddleware` as nil, so it might be good to implement it or use a default for the CLI if `GetFunctionRunJobs` is called, which it could be.
 https://github.com/inngest/inngest/blob/b84a0e5fe2437e722145ca0938ffedb6dec39701/pkg/devserver/devserver.go#L389-L396
https://github.com/inngest/inngest/blob/b84a0e5fe2437e722145ca0938ffedb6dec39701/pkg/devserver/devserver.go#L389-L396

Reply:
Oh funny, somehow this didn't get suggested in my IDE -- will fix!